packages/build-config/configs/node.js

```diff
@@ -23,9 +23,10 @@ function findNodeModules(start) {
 
 function shouldIncludeExternalModuleInBundle(module) {
   return (
+    module.indexOf('hops') === 0 ||
     module.indexOf('core-js') === 0 ||
     module.indexOf('babel-polyfill') === 0 ||
-    module.indexOf('hops') === 0 ||
+    /^.*\.(?!js(?:on)?$)/.test(module) ||
```

Comment:
We should also add `.mjs` and maybe even `.node`.
Also there should be a comment, explaining, that this is based on the Node.js resolve algorithm: https://nodejs.org/api/modules.html#modules_all_together